README.md

```diff
@@ -120,6 +120,31 @@ Where the `formats` available are:
 
 Note that flags using the @<version> can be used for earlier versions of each specification as well.
 
+## Format conversion (experimental)
```

Comment:
Most of the content was moved from the main README to the wiki in order to help focus new users on the specific needs to get started. Instead of adding this, we should probably update to point to this page on the wiki: https://github.com/anchore/syft/wiki/format-conversion

Reply:
Alright, on my way to fix it, should I delete this pull request? I couldn't find this on the wiki page and I don't think I have permissions to edit that.

Reply:
It was hidden on the wrong page. I moved that content to its own top-level wiki page just now, as it should have been.